File: Jack_Charris/Programas/Dart/drenta.dart
import 'dart:io';

void main() {
  print("Ingrese su nombre de usuario:");
  String usuario = stdin.readLineSync()!;

  print("Ingrese su contraseña:");
  String password = stdin.readLineSync()!;

  if ( password.isNotEmpty ) {
    print("Bienvenido $usuario");
  }

  print("Ingrese el monto de la renta");
  double monto = double.parse(stdin.readLineSync()!);

  if (monto >= 0 && monto <= 10) {

    double descuento = monto * 0.2;
    double total = monto - descuento;
    print(" Hola $usuario, usted tiene un descuento del 20%");
    print("El valor total es: $total");

  } else if (monto >= 11 && monto <= 100) {

    double descuento = monto * 0.1;
    double total = monto - descuento;
    print(" Hola $usuario, usted tiene un descuento del 10%");
    print("El valor total es: $total");

  } else if (monto >= 101 && monto <= 1000) {

    double descuento = monto * 0.05;
    double total = monto - descuento;
    print(" Hola $usuario, usted tiene un descuento del 5%");
    print("El valor total es: $total");

  } else if (monto > 1000) {

    double descuento = monto * 0.01;
    double total = monto - descuento;
    print(" Hola $usuario, usted tiene un descuento del 1%");
    print("El valor total es: $total");

  }
}

File: Jack_Charris/Programas/Ruby/drenta.rb
puts "Ingrese su nombre de usuario:"
## nombre
   usuario = gets.chomp 

puts "Ingrese su contraseña"
    password = gets.chomp

puts "Ingrese el monto de la renta"
    monto = gets.chomp.to_f

    if monto >= 0 && monto <= 10

    descuento = monto * 0.2
    total = monto - descuento
    puts " Hola #{usuario}, usted tiene un descuento del 20%"
    puts "El valor total es: #{total}"

    end

   if monto >= 11 && monto <= 100

    descuento = monto * 0.1
    total = monto - descuento
    puts " Hola #{usuario}, usted tiene un descuento del 10%"
    puts "El valor total es: #{total}"
     
end

   if monto >= 101 && monto <= 1000

    descuento = monto * 0.05
    total = monto - descuento
    puts " Hola #{usuario}, usted tiene un descuento del 5%"
    puts "El valor total es: #{total}"
     
end

    if (monto > 1000) 

    descuento = monto * 0.01
    total = monto - descuento
    puts " Hola #{usuario}, usted tiene un descuento del 1%"
    puts "El valor total es: #{total}"

    end


  